README.md

```diff
@@ -56,7 +56,9 @@ A curated list of awesome Direct Volume Rendering articles, software and resourc
 
 ### Ultrasound
 
-- **The ultrasound visualization pipeline** (2014), Å. Birkeland [pdf](https://pdfs.semanticscholar.org/b713/ccf2c88e73e951489cc89ecc1381a7e425fd.pdf)
+- **The ultrasound visualization pipeline** (2014), Å. Birkeland et al. [pdf](https://pdfs.semanticscholar.org/b713/ccf2c88e73e951489cc89ecc1381a7e425fd.pdf)
+- **Real-Time Processing for Advanced Ultrasound Visualization** (2016), Schulte zu Berge, Christian Ulrich, Nassir Navab, Nassir Navab, and Bernhard Preim. [pdf](https://mediatum.ub.tum.de/doc/1285497/file.pdf) 
+- **Feature Emphasis and Contextual Cutaways for Multimodal Medical Visualization** (2007), M. Burns et al. [pdf](https://gfx.cs.princeton.edu/pubs/Burns_2007_FEA/contextual_cutaways.pdf)
```

Comment:
There is just one thing that is bugging me:
Is there a reason why the publications are not sorted by publication year ?

Reply:
Nope, that's a good suggestion. More recent first I guess ?

Reply:
It seems that generally least recent first is more used in the current list (There are some mixup at some point) :smile:

Reply:
It really think it makes more sense to sort from most recent to least recent... I applied that on all publications.